kpm/lib/kpm/installer.rb

```diff
@@ -27,13 +34,12 @@ def self.build_default_config(all_kb_versions=nil)
         next if version.nil?
 
         major, minor, patch, pre = version.segments
-        next if !pre.nil? || minor.nil? || minor.to_i.odd?
+        next if !pre.nil? || minor.nil? || minor.to_i.odd? || patch.to_i.odd?
```

Comment:
I was confused. I was in the assumption that only even numbers formed a stable version.